File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    babb = ["aya","ye","woo","ma"]
    for ba in babbling:
        for i in babb:
            ba = ba.replace(i, '!',1)
        ba = ba.replace('!', '')
        if len(ba) == 0 :
            answer += 1
    return answer

File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    var answer = 0;
    babblings = ['aya','ye','woo','ma']
    for (let i in babbling){
        let init = babbling[i]
        for(let j in babblings){
            if(init.includes(babblings[j])){
                init = init.replace(babblings[j], "X")
            }
        }
        init = init.replace(/X/gi,"")
        
        if(init.length === 0){
            answer += 1
        }
    } 
    return answer;
}